renormalizer/transport/kubo.py

```diff
@@ -233,21 +247,22 @@ def init_mps(self):
             return BraKetPair(bra_mpdm, ket_mpdm, self.j_oper), BraKetPair(bra_mpdm, ket_mpdm2, self.j_oper2)
 
     def process_mps(self, mps):
+        # add the negative sign because `self.j_oper` is taken to be real
```

Comment:
The -1 here is due to -1.0j * -1.0j of the two j_operators, right?
If so, why don't keep the -1.0j factor to the j_operator since during the time propagation, the mps is complex anyway.
The -1 here looks strange.

Reply:
Yes. j_operator is taken to be real for better performance. In my practice, it is sometimes found that J|ket(0)> is very time and memory consuming. Because |ket(0)> is real, a real j_operator could save half memory and computational cost.

Reply:
I see.